src/app/pages/api-keys/store/api-key.effects.ts

```diff
@@ -0,0 +1,47 @@
+import { Injectable } from '@angular/core';
+import { Actions, createEffect, ofType } from '@ngrx/effects';
+import { TranslateService } from '@ngx-translate/core';
+import { of } from 'rxjs';
+import {
+  catchError, filter, map, switchMap,
+} from 'rxjs/operators';
+import { ApiEventMessage } from 'app/enums/api-event-message.enum';
+import { WebSocketService } from 'app/services';
+import {
+  apiKeyRemoved, apiKeysLoaded, apiKeysNotLoaded, apiKeysPageEntered,
+} from './api-key.actions';
+
+@Injectable()
```

Comment:
I think having fully fledged ngrx store for api keys is too much.
ngrx store is meant more for global things, like users and system configuration.
In here, just having a component store should be enough.

Reply:
Yeah, good point 👍🏻